CMakeLists.txt

```diff
@@ -106,7 +106,7 @@ endif()
 # that the allocation from MbedTLS is < 4k, 4033bytes to be precise.  We can
 # also change the input buffer length, but this requires that all communication
 # partners support the TLS protocol extension to tell them about this.
-set(MBEDTLS_C_FLAGS "-DMBEDTLS_SSL_IN_CONTENT_LEN=4608 -DMBEDTLS_SSL_OUT_CONTENT_LEN=3700 -DMBEDTLS_PLATFORM_MEMORY=1")
+set(MBEDTLS_C_FLAGS "-DMBEDTLS_SSL_IN_CONTENT_LEN=6000 -DMBEDTLS_SSL_OUT_CONTENT_LEN=6000 -DMBEDTLS_PLATFORM_MEMORY=1")
```

Comment:
Is this change still necessary? If so, you should definitely update the comment above.